CMakeLists.txt

```diff
@@ -200,8 +200,15 @@ SET(QT_QTTEST_LIBRARY Qt5::Test)
 
 # check for libsndfile
 FIND_PACKAGE(SndFile REQUIRED)
-IF(NOT SNDFILE_FOUND)
-	MESSAGE(FATAL_ERROR "LMMS requires libsndfile1 and libsndfile1-dev >= 1.0.18 - please install, remove CMakeCache.txt and try again!")
+IF(SNDFILE_FOUND)
+  IF(SNDFILE_VERSION VERSION_GREATER_EQUAL 1.1.0)
+    SET(LMMS_SNDFILE_MP3 TRUE)
+  ELSE()
+    MESSAGE("libsndfile version is < 1.1.0; MP3 import disabled")
+    SET(LMMS_SNDFILE_MP3 FALSE)
```

Comment:
Should this be `LMMS_HAVE_MP3IMPORT`?

Reply:
Or I guess `LMMS_HAVE_MP3IMPORT` would need to be set in `lmmsconfig.h`?

Reply:
Also, please note that Setting `LMMS_SNDFILE_MP3` as a CMake variable doesn't have an effect. To expose it to C++ code, you should `#cmakedefine` it in `src/lmmsconfig.h.in`.